js/js_bench_test.go

```diff
@@ -0,0 +1,31 @@
+package js_test
+
+import (
+	"testing"
+
+	"github.com/sensu/sensu-go/js"
+	"github.com/sensu/sensu-go/types"
+	"github.com/sensu/sensu-go/types/dynamic"
+)
+
+func BenchmarkCheckEval(b *testing.B) {
+	check := types.FixtureCheck("foo")
+	for i := 0; i < b.N; i++ {
+		synth := dynamic.Synthesize(check)
+		params := map[string]interface{}{
+			"check": synth,
+		}
```

Comment:
Shouldn't `synth` and `params` be defined outside the loop to not skew the benchmark (since they don't depend on `i` and are not part of the function you want to measure, here `Evaluate()`)?

Reply:
No, because a typical filter invocation will require a synthesis, and I wanted to measure worst case performance.